CONTRIBUTING.md

```diff
@@ -72,38 +72,38 @@ $ git push origin [name_of_your_new_branch]
 
 ## Introducing New Features
 
-We would 💖 you to contribute to Framework, but we would also like to make sure Framework is as great as possible and loyal to its vision and mission statement 🙏.
+We would 💖 you to contribute to Utopia-PHP HTTP, but we would also like to make sure Http is as great as possible and loyal to its vision and mission statement 🙏.
 
 For us to find the right balance, please open an issue explaining your ideas before introducing a new pull request.
 
-This will allow the Framework community to have sufficient discussion about the new feature value and how it fits in the product roadmap and vision.
+This will allow the Utopia-PHP Http community to have sufficient discussion about the new feature value and how it fits in the product roadmap and vision.
```

Comment:
```suggestion
This will allow the community to have sufficient discussion about the new feature value and how it fits in the product roadmap and vision.
```